src/components/OptionPage.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import ImageItem from "./ImageItem";
+import CardGeneralSettings from "./CardGeneralSettings";
+import CardAbout from "./CardAbout";
+
+class OptionPage extends React.Component {
+    constructor() {
+        super(React.Component);
+    }
+
+    render() {
+        return (
+            <div className="d-flex flex-column align-items-center justify-content-center">
+                <ImageItem image="./images/promo.png"></ImageItem>
```

Comment:
How about we imitate the [documentation](https://facebook.github.io/create-react-app/docs/adding-images-fonts-and-files) and do `import promo from './images/promo.png';`?